### PR TITLE
Translated Euclid's Algorithm example to English

### PR DESCRIPTION
**Pull request by kortenkamp**
The example is still showing German, this fixes it. The Headline should be changed as well, but this will be in a separate pull request.

### CODE REVIEW

File: src/gallery/main/EuclidsAlgorithm/EuclidsAlg.html [hunk 1]
```diff
@@ -93,7 +93,7 @@
 
                            );
 
-                     drawtext((65,43-i*7),a+" ist der ggT",size->20,color->(0.6,0,0));
+                     drawtext((65,43-i*7),a+" ist the gcd",size->20,color->(0.6,0,0));
```
Comment (gagern) [inline src/gallery/main/EuclidsAlgorithm/EuclidsAlg.html]: You forgot “ist”.

Comment (kortenkamp) [inline src/gallery/main/EuclidsAlgorithm/EuclidsAlg.html]: that was autocorrect... great.

**Comment by kortenkamp** [comment]
No idea how I append this further change in GitHub. Please advise.

**Comment by gagern** [comment]
Probably by editing https://github.com/CindyJS/website/blob/euclid's-algorithm/src/gallery/main/EuclidsAlgorithm/EuclidsAlg.html i.e. the version of the file in the `euclid's-algorithm` branch you created. You can switch branches using the dropdown in the top left area of the code or directory listing.

File: src/gallery/main/EuclidsAlgorithm/EuclidsAlg.html (hunk 1 above)
Comment (kortenkamp) [inline src/gallery/main/EuclidsAlgorithm/EuclidsAlg.html]: Ok, fixed that.

**Pull request merged**